Task2/Index.html

```diff
@@ -0,0 +1,76 @@
+    </div>
+
+    <div id="a2">
```

Comment:
Usually for creating anchor is used attribute `name` of tag `<a>`. Hint: you can use empty `<a>` element with an attribute `name`. Read more [here](http://htmlbook.ru/html/a)